### PR TITLE
Fix Del Sgr distance; update Alf Cen names for consistency with IAU

### PR DESCRIPTION
**Pull request by SevenSpheres**
Some things that were forgotten in #25...

### CODE REVIEW

**Comment by SevenSpheres** [comment]
Using tabs was the standard in the old revised.stc

**Comment by 375gnu** [comment]
okay, let's live with it

File: data/revised.stc [hunk 1]
```diff
@@ -180,27 +180,33 @@ Modify 82257 # Improved spectral class from McCook & Sion
...
 }
 
 Modify 88267 # 95 Her: Spectral class from SIMBAD; replaces "G5"
 {
 	SpectralType "A5III"
 }
 
+Modify 89931 # Del Sgr: Distance from Hipparcos parallax, which is more accurate than the Gaia parallax
+{
+	Distance 347.726 # 9.38 +/- 0.18 mas
```
Comment (LukeCEL) [inline data/revised.stc:206]: Using a parallax of 9.38 mas (and the "lazy" inversion method) I got 347.715 ly. I know this is a small difference but it's probably best to clear this up to minimize confusion.

Comment (SevenSpheres) [reply data/revised.stc:206]: I used a parsecs-to-light-years conversion factor of 3.26167, which IIRC is Celestia's internal value.

**Comment by SevenSpheres** [comment]
> Can you also fix the distance of del2 Lyr (HIP 92791)?

Done

File: data/revised.stc (hunk 1 above)
Comment (LukeCEL) [reply data/revised.stc:206]: Okay, that makes sense. But I wonder why would Celestia use 3.26167? The correct value (when rounded) would be 3.26156. Very strange.

**Pull request merged**